recipes/sktime/LICENSE

```diff
@@ -0,0 +1,30 @@
+BSD 3-Clause License
```

Comment:
You do not need to manually include a copy of this file since it comes in the source distribution. Please remove it